cmd/main.go

```diff
@@ -473,7 +474,7 @@ func setupDB(log logrus.FieldLogger) *gorm.DB {
 func getOCMClient(log logrus.FieldLogger, metrics metrics.API) *ocm.Client {
 	var ocmClient *ocm.Client
 	var err error
-	if Options.Auth.EnableAuth {
+	if Options.Auth.ResolvedAuthType() == auth.TypeRHSSO {
```

Comment:
Do we expect to introduce additional auth types? Maybe worth having a switch then?

Reply:
Not in this case. I think we will only use the ocm client when we're authenticating against Red Hat SSO